### DISCUSSION TITLE
Add a Val Town guide

### DISCUSSION THREAD

**Discussion by stevekrouse** [ardatan/feTS#1825] [Ideas]
**Is your feature request related to a problem? Please describe.**

I am a founder of https://val.town and we can deploy feTS!

**Describe the solution you'd like**

I'd like to contribute a guide via a pull request in the spirit of [this deno one](https://the-guild.dev/openapi/fets/server/integrations/deno), as well as be included in the "Deploy Anywhere" section on the homepage.

**Additional context**

Here's a demo of a feTS server on Val Town: https://www.val.town/v/pomdtr/fets

**Comment by stevekrouse** [comment]
I wanted to create this ticket before sending over an unsolicited PR. If you're open to this, please let me know and I'll send a PR!

**Comment by ardatan** [comment]
Of course! Feel free to create! This looks so cool!
